SmartEnrollmentFor911/SmartEnrollmentFor911/Pages/Index.cshtml.cs

```diff
@@ -13,24 +13,21 @@ namespace SmartEnrollmentFor911.Pages
 {
     public class IndexModel : PageModel
     {
-        private readonly ILogger<IndexModel> _logger;
+        //private readonly ILogger<IndexModel> _logger;
 
-        public IndexModel(ILogger<IndexModel> logger)
-        {
-            _logger = logger;
-        }
+        //public IndexModel(ILogger<IndexModel> logger)
+        //{
+        //    _logger = logger;
+        //}
 
         public void OnGet()
         {
             using (var webClient = new WebClient())
             {
-                String crimeIncidentJSON = webClient.DownloadString("https://data.cincinnati-oh.gov/resource/k59e-2pvf.json");
-                var crimeIncidents = CrimeIncidents.FromJson(crimeIncidentJSON);
+                var crimeIncidents = CrimeIncidents.FromJson(GetJsonData("https://data.cincinnati-oh.gov/resource/k59e-2pvf.json"));
```

Comment:
I'd stick with the old version.  While consolidating to one line saves space, it's much more difficult to identify the root cause of an error when the error indicates line 27, and you have multiple method calls on line 27.